doc/doxygen/Doxyfile.in

```diff
@@ -753,7 +753,7 @@ SHOW_NAMESPACES        = YES
 # The FILE_VERSION_FILTER tag can be used to specify a program or script that
 # doxygen should invoke to get the current version for each file (typically from
 # the version control system). Doxygen will invoke the program by executing (via
-# popen()) the command command input-file, where command is the value of the
+# popen()) the command input-file, where command is the value of the
```

Comment:
I'm not particularly certain about this change, and it's possible the file is from upstream doxygen and shouldn't be touched.